src/dp_flow.c

```diff
@@ -199,6 +204,7 @@ int dp_build_flow_key(struct flow_key *key /* out */, struct rte_mbuf *m /* in *
 	default:
 		key->port_dst = 0;
 		key->src.port_src = 0;
+		// should this not return DP_ERROR?
 		break;
 	}
```

Comment:
These two I actually just thought I would return and see what happens, as I do not really know the implications right now

Reply:
What do you mean if key->l3_type == IPV6 ?
There is such a line before the switch `key->l3_type = df->l3_type;` and in this default branch of switch, it is impossible that it can be IPv6 as it would be another branch of the switch.
Returning error should be fine as it would  drop the packet and it will probably make us more robust against fuzzed packets during parsing.